source/tyr/tyr/validations.py

```diff
@@ -1,8 +1,18 @@
 from datetime import datetime
-import pytz
+import geojson
+import json
 
 def datetime_format(value):
     """Parse a valid looking date in the format YYYYmmddTHHmmss"""
 
     return datetime.strptime(value, "%Y%m%dT%H%M%SZ")
 
+def json_format(value):
+
+    if value:
+        features= value.get('features')
```

Comment:
what happen if I post a Feature (not a FeatureCollection) ?
```
{
  "type": "Feature",
  "geometry": {
    "type": "Point",
    "coordinates": [125.6, 10.1]
  },
  "properties": {
    "name": "Dinagat Islands"
  }
}
```
This geojosn has no attribute "features"
[http://geojson.org/](http://geojson.org/)